<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLSettingsUnitWidget</class>
 <widget class="QWidget" name="qMRMLSettingsUnitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>121</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="UnitLabel">
     <property name="text">
      <string>Unit:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="UnitNodeComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Select the current unit node to modify.</string>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLUnitNode</string>
      </stringlist>
     </property>
     <property name="showHidden">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="qMRMLUnitWidget" name="UnitInfoWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLUnitWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLUnitWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
